src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js

```diff
@@ -188,6 +196,20 @@ const StatusForm = ({ defaultTexts, incident, onClose }) => {
               rows={10}
             />
 
+            <Notification warning data-testid="statusFormToelichting">
+              {isDeelmelding ? DEELMELDING_EXPLANATION : MELDING_EXPLANATION}
+            </Notification>
+
+            {!isDeelmelding && (
+              <FieldControlWrapper
+                control={form.get('send_email')}
+                data-testid="statusFormSendEmailField"
+                name="send_email"
+                label={MELDING_CHECKBOX_DESCRIPTION}
+                render={CheckboxInput}
+              />
```

Comment:
There doesn't seem to be a test for this element

Reply:
done